projects/components/src/filtering/filter/builder/types/abstract-filter-builder.ts

```diff
@@ -19,6 +19,14 @@ export abstract class AbstractFilterBuilder<TValue extends FilterValue> {
     return [...this.supportedTopLevelOperators(), ...this.supportedSubpathOperators()];
   }
 
+  public buildFiltersForSupportedSubpathOperators(
+    attribute: FilterAttribute,
+    value: TValue,
+    subpath: string
```

Comment:
I suspect you'll end up with much cleaner code if you fix FilterAttribute to reflect that an attribute may include a subpath rather than trying to move them around in tandem. That said, haven't looked at the filter button stuff in depth, so it's possible it needs a more thorough refactor to fix that and it's out of scope - I'd at least suggest giving it a try though.